components/AudioControls.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <v-container fluid>
+    <v-row>
+      <v-col cols="9" md="3" class="d-flex flex-row pa-0">
+        <v-avatar ref="albumCover" tile size="72" color="primary">
+          <v-img :src="getImageUrl(getCurrentItem.AlbumId)">
+            <template #placeholder>
+              <v-icon dark>mdi-album</v-icon>
+            </template>
+          </v-img>
+        </v-avatar>
+        <div class="d-flex flex-column justify-center ml-4">
+          <span class="font-weight-medium mt-md-n2">
+            <nuxt-link
+              tag="span"
+              class="text-truncate link"
+              :to="`/item/${getCurrentItem.AlbumId}`"
+              >{{ getCurrentItem.Name }}</nuxt-link
+            >
+            <v-btn class="d-none d-md-inline-flex" icon>
+              <v-icon size="18">{{
+                getCurrentItem.UserData.IsFavorite
+                  ? 'mdi-heart'
+                  : 'mdi-heart-outline'
+              }}</v-icon>
+            </v-btn>
+          </span>
+          <nuxt-link
+            tag="span"
+            class="text--secondary text-caption text-truncate mt-md-n2 link"
+            :to="`/artist/${getCurrentItem.AlbumArtists[0].Id}`"
+          >
+            {{ getCurrentItem.AlbumArtist }}
+          </nuxt-link>
+        </div>
+      </v-col>
+      <v-col cols="6" class="pa-0 d-none d-md-inline">
+        <div class="d-flex flex-column justify-center">
+          <div class="d-flex align-center justify-center">
+            <v-btn disabled icon class="mx-1">
+              <v-icon>mdi-shuffle</v-icon>
+            </v-btn>
+            <v-btn icon class="mx-1" @click="setPreviousTrack">
+              <v-icon>mdi-skip-previous</v-icon>
+            </v-btn>
+            <v-btn icon large class="mx-1" @click="togglePause">
+              <v-icon large>
+                {{ isPaused ? 'mdi-play' : 'mdi-pause' }}
+              </v-icon>
+            </v-btn>
+            <v-btn icon class="mx-1" @click="stopPlayback">
+              <v-icon>mdi-stop</v-icon>
+            </v-btn>
+            <v-btn icon class="mx-1" @click="setNextTrack">
+              <v-icon>mdi-skip-next</v-icon>
+            </v-btn>
+            <v-btn disabled icon class="mx-1">
+              <v-icon>mdi-repeat-off</v-icon>
+            </v-btn>
+          </div>
+          <v-slider
+            :value="sliderValue"
+            hide-details
+            min="0"
+            :max="runtime"
+            validate-on-blur
+            thumb-label
```

Comment:
```suggestion
```
The thumb label shows the incorrect values, since the slider stores value in terms of seconds

Reply:
@Camc314 They're overriden in the template below

Reply:
Ahh I missed that, nice